### PR TITLE
WIP: fix(@schematics/angular): add ServiceWorkerModule in dev mode, but disabled

### PR DESCRIPTION
**Pull request by alxhub**
This PR requires https://github.com/angular/angular/pull/20720 to be released before it should be merged.

### CODE REVIEW

**Comment by hansl** [comment]
This is blocked on a release of Angular 5.1.

**Comment by alxhub** [comment]
This is now unblocked and can be merged. @hansl

File: packages/schematics/angular/application/other-files/app.module.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { environment } from '../environments/environment';<% } %>
   imports: [
     BrowserModule<% if (routing) { %>,
     AppRoutingModule<% } %><% if (serviceWorker) { %>,
-    environment.production ? ServiceWorkerModule.register('/ngsw-worker.js') : []<% } %>
+    ServiceWorkerModule.register('/ngsw-worker.js', {enabled: environment.production})<% } %>
```
Comment (hansl) [inline packages/schematics/angular/application/other-files/app.module.ts]: Only comment: spaces between `{}` and its content.

Comment (alxhub) [reply packages/schematics/angular/application/other-files/app.module.ts]: Done.

**Pull request merged**